packages/dev/core/test/unit/Math/babylon.math.vector.test.ts

```diff
@@ -247,7 +247,7 @@ describe("Babylon Vector tests", () => {
             it("should normalize the vector", () => {
                 vector1.normalize();
                 const length = vector1.length();
-                expect(length).toBeCloseTo(1);
+                expect(length).toBeCloseTo(2.23606 /* sqrt(5) */);

```

Comment:
This should not have change, or it will create a lot of backward incompatibility.

We know it is not consistent but unfortunately back compat first :-)

Reply:
Fixed in [157fc18](https://github.com/BabylonJS/Babylon.js/pull/13699/commits/157fc18cd530c9949d920475cb52ee5670b09539)